### PR TITLE
fix: add design fixes and api calls reduce in leaderboard

### LINKED ISSUE #379: Put the top 3 profil clickable on leaderboard
Have the ability to click on top 3 card and see their profil

Screenshot 2023-11-30 at 15 12 00

### PR DESCRIPTION
**Pull request by ayushtom**
closes #379 #380 #408

### CODE REVIEW

File: app/leaderboard/page.tsx [hunk 1]
```diff
@@ -65,12 +63,16 @@ export default function Page() {
...
+    setTimeout(() => {
+      setApiCallDelay(true);
+    }, 1000);
     if (address === "") return;
     if (address) setUserAddress(address);
     if (status === "disconnected") setUserAddress("");
   }, [address, status]);
 
   useEffect(() => {
+    if (!apiCallDelay) return;
```
Comment (Th0rgal) [inline app/leaderboard/page.tsx:76]: Please add a comment to explain why you are adding a 1sec delay.

**Review by Th0rgal — CHANGES_REQUESTED** [review]

File: app/leaderboard/page.tsx (hunk 1 above)
Comment (ayushtom) [reply app/leaderboard/page.tsx:76]: done

File: services/apiService.ts [hunk 2]
```diff
@@ -15,10 +17,12 @@ const baseurl = process.env.NEXT_PUBLIC_API_LINK;
 export const fetchLeaderboardToppers = async (
   params: LeaderboardTopperParams
 ) => {
+  console.log({ params });
```
Comment (fricoben) [inline services/apiService.ts]: to delete

File: services/apiService.ts [hunk 3]
```diff
@@ -15,10 +17,12 @@ const baseurl = process.env.NEXT_PUBLIC_API_LINK;
...
   try {
-    const { addr } = params;
+    const { addr, start_timestamp, end_timestamp } = params;
+    console.log({ params });
```
Comment (fricoben) [inline services/apiService.ts]: to delete also

**Review by fricoben — CHANGES_REQUESTED** [review]
some console.log to delete

File: services/apiService.ts (hunk 2 above)
Comment (ayushtom) [reply services/apiService.ts]: done

File: services/apiService.ts (hunk 3 above)
Comment (ayushtom) [reply services/apiService.ts]: done

**Review by Th0rgal — APPROVED** [review]

**Review by fricoben — APPROVED** [review]
lgtm

**Pull request merged**